src/structures/member.ts

```diff
@@ -1,26 +1,28 @@
-import cache from '../models/cache.ts'
 import { Client } from '../models/client.ts'
 import { MemberPayload } from '../types/guild.ts'
 import { Base } from './base.ts'
+import { Role } from './role.ts'
 import { User } from './user.ts'
 
 export class Member extends Base {
   id: string
   user: User
   nick?: string
-  roles: string[]
+  roleIDs: string[]
+  roles: Role[] = []
```

Comment:
I think this property can be removed

Reply:
Cannot be yet, it is used to retrieve role objects. In next PRs I'll add MemberRolesManager.